### ISSUE TITLE
panic when using MSSQL as source

### ISSUE THREAD

**Issue by creativedutchmen** [slingdata-io/sling-cli#7]
When using an Azure MSSQL database as the source, running a sync fails with:

```
8:14AM INF connecting to source database (azuresql)
8:14AM INF reading from source database
panic: Not implemented

goroutine 37 [running]:
github.com/denisenkom/go-mssqldb.passthroughConn.SetWriteDeadline(...)
	/home/runner/go/pkg/mod/github.com/denisenkom/go-mssqldb@v0.9.0/net.go:167
crypto/tls.(*Conn).SetWriteDeadline(...)
	/opt/hostedtoolcache/go/1.19.12/x64/src/crypto/tls/conn.go:152
crypto/tls.(*Conn).closeNotify(0xc000d3e000)
	/opt/hostedtoolcache/go/1.19.12/x64/src/crypto/tls/conn.go:1399 +0xdb
crypto/tls.(*Conn).Close(0x0?)
	/opt/hostedtoolcache/go/1.19.12/x64/src/crypto/tls/conn.go:1369 +0x69
github.com/denisenkom/go-mssqldb.(*Conn).Close(0xc000249af0?)
	/home/runner/go/pkg/mod/github.com/denisenkom/go-mssqldb@v0.9.0/mssql.go:361 +0x28
database/sql.(*driverConn).finalClose.func2()
	/opt/hostedtoolcache/go/1.19.12/x64/src/database/sql/sql.go:645 +0x3c
database/sql.withLock({0x2687988, 0xc0009ec6c0}, 0xc000249ba8)
	/opt/hostedtoolcache/go/1.19.12/x64/src/database/sql/sql.go:3439 +0x8c
database/sql.(*driverConn).finalClose(0xc0009ec6c0)
	/opt/hostedtoolcache/go/1.19.12/x64/src/database/sql/sql.go:643 +0x116
database/sql.(*DB).Close(0xc0005fc340)
	/opt/hostedtoolcache/go/1.19.12/x64/src/database/sql/sql.go:903 +0x297
github.com/flarco/dbio/database.(*BaseConn).Close(0xc00077d6c0)
	/home/runner/go/pkg/mod/github.com/flarco/dbio@v0.3.257/database/database.go:664 +0x34
github.com/slingdata-io/sling-cli/core/sling.(*TaskExecution).runDbToFile(0xc000218b40)
	/home/runner/work/sling-cli/sling-cli/core/sling/task_run.go:262 +0x528
github.com/slingdata-io/sling-cli/core/sling.(*TaskExecution).Execute.func1()
	/home/runner/work/sling-cli/sling-cli/core/sling/task_run.go:87 +0x259
created by github.com/slingdata-io/sling-cli/core/sling.(*TaskExecution).Execute
	/home/runner/work/sling-cli/sling-cli/core/sling/task_run.go:68 +0x41b
```

Running in Docker (latest, 1.0.15) on M1 Mac

**Comment by creativedutchmen** [comment]
Seems to be related to https://github.com/denisenkom/go-mssqldb/issues/639

**Comment by flarco** [comment]
Hey, thanks for opening an issue.
I updated the `go-mssqldb` to its latest version and released Sling version `v1.0.17`.
Can you try it?
